.travis.yml

```diff
@@ -1,10 +1,21 @@
-sudo: required
+sudo: false
 language: c
 os:
   - linux
 branches:
   except:
     - nightly
+addons:
+    apt_packages:
+        - xclip
+        - gdb
+        - valgrind
+        - libtool
+        - gcc-multilib
```

Comment:
Am I wrong in assuming that `gcc-multilib` will be installed for all architectures?